docs/api_status.md

```diff
@@ -202,7 +202,23 @@ with respect to Memcached and Redis APIs.
   - [ ] PFMERGE
 
 ### API 3
+- [X] Generic Family
+  - [ ] TOUCH 
+- [X] HashSet Family
+  - [X] HSTRLEN
+- [X] Server Family
+  - [ ] CLIENT REPLY
+  - [X] REPLCONF
+  - [ ] WAIT
+
 ### API 4
+
+- [X] Generic Family
+  - [X] UNLINK
+- [X] Server Faimily
```

Comment:
Why is this added if non are checked?

Reply:
To show commands that are not supported, like in HYPERLOGLOG or STREAM for API 6